tests/eeid_plugin/test_helpers.c

```diff
@@ -24,7 +24,7 @@ oe_result_t make_test_eeid(oe_eeid_t** eeid, size_t data_size)
     for (size_t i = 0; i < data_size; i++)
         (*eeid)->data[i] = 'a' + (i % 26);
     (*eeid)->data[data_size - 1] = 0;
-    (*eeid)->size_settings.num_heap_pages = 100 + (data_size / OE_PAGE_SIZE);
+    (*eeid)->size_settings.num_heap_pages = 110 + (data_size / OE_PAGE_SIZE);
```

Comment:
Why is this needed?

Reply:
For some reason the latest master needs a bit more memory, I didn't investigate it any deeper, but without that it ran out of heap space.